src/Mod/Draft/Draft.py

```diff
@@ -1809,17 +1811,23 @@ def rotate(objectslist,angle,center=Vector(0,0,0),axis=Vector(0,0,1),copy=False)
                 newobj.ViewObject.Rotation = -angle
         elif getType(obj) == "Point":
             v = Vector(obj.X,obj.Y,obj.Z)
-            rv = v.sub(center)
-            rv = DraftVecUtils.rotate(rv,math.radians(angle),axis)
-            v = center.add(rv)
+            rv = v.sub(c)
+            rv = DraftVecUtils.rotate(rv,math.radians(angle),a)
+            v = c.add(rv)
```

Comment:
I suggest using more descriptive names, instead of just `c`, `a`. I prefer the older names `center`, `axis`, etc.